File: skills/quality/dry-test/fixtures/go/clone.go
package fixture

import "strconv"

func Alpha(value int) string {
	// Only Alpha carries a comment. Comments are spelling, so Beta must still match it.
	label := "alpha: "
	total := value + 1
	if total > 10 {
		return label + strconv.Itoa(total)
	}
	return label
}

func Beta(number int) string {
	tag := "beta: "
	amount := number + 2
	if amount > 20 {
		return tag + strconv.Itoa(amount)
	}
	return tag
}

File: skills/quality/dry-test/fixtures/rust/clone.rs
fn alpha(value: i32) -> String {
    // Only alpha carries a comment. Comments are spelling, so beta must still match it.
    let label = String::from("alpha: ");
    let total = value + 1;
    if total > 10 {
        return label + &total.to_string();
    }
    label
}

fn beta(number: i32) -> String {
    let tag = String::from("beta: ");
    let amount = number + 2;
    if amount > 20 {
        return tag + &amount.to_string();
    }
    tag
}
